### PR TITLE
Make ValidatedNumerics.jl into a meta-package

### PR DESCRIPTION
**Pull request by dpsanders**
This PR makes ValidatedNumerics.jl a "meta-package" that loads and reexports the main JuliaIntervals packages (currently there are 3 of them).

The tests just do basic sanity checks that the packages load and have basic functionality working; detailed tests are left to the individual packages.

### CODE REVIEW

**Comment by dpsanders** [comment]
This is ready for comments, @lbenet.

File: README.md [hunk 1]
```diff
@@ -17,10 +17,21 @@ To install the package, from within Julia do
...
 ## Interval arithmetic
 All calculations are carried out using **interval arithmetic**: all quantities are treated as intervals, which are propagated throughout a calculation. The final result is an interval that is *guaranteed* to contain the correct result, starting from the given initial data.
 
-The aim of the package is correctness over speed, although performance considerations are also taken into account
+The aim of the package is correctness, together with performance.
+
+
 
 ## Documentation
 Documentation is available [**here**](http://dpsanders.github.io/ValidatedNumerics.jl/).
```
Comment (lbenet) [inline README.md]: This link should be changed to https://juliaintervals.github.io/ValidatedNumerics.jl/

Comment (lbenet) [inline README.md]: ... though it currently includes all the documentation put together. Probably we should simply merge this and the update the documentation (before tagging v0.9).

**Review by lbenet — COMMENTED** [review]
I think there is no much to say on this, except for the documentation (and maybe wait for https://github.com/JuliaIntervals/IntervalArithmetic.jl/pull/20 to be merged).

File: README.md (hunk 1 above)
Comment (dpsanders) [inline README.md]: Link fixed, thanks. I agree, let's merge and then update the docs.

**Comment by lbenet** [comment]
I'm merging, despite of the failing tests!

**Comment by lbenet** [comment]
Thanks!

**Pull request merged**